ocean_lib/test/test_exceptions.py

```diff
@@ -34,34 +34,3 @@ def test_InsufficientBalance(publisher_ocean_instance, publisher_wallet):
             alice,
             alice.address,
         )
-
-
-def test_ContractNotFound(publisher_ocean_instance, metadata, publisher_wallet):
```

Comment:
Are these exceptions still valid to use? Or it's not necessary to test them?

Reply:
I've moved the tests in the ocean_assets class, with the other asset creation tests. Both these exceptions came from there.